src/main/resources/application.yml

```diff
@@ -9,7 +9,7 @@ spring:
     password: ${DB_PASSWORD:password}
     username: ${DB_USERNAME:sa}
     url: ${DB_URL:jdbc:h2:mem:testdb}
-  h2.console.enabled: false
+  h2.console.enabled: true
```

Comment:
Like this for the testing version of the appliation.yml.  In the root, is this a security issue?

Reply:
Agreed. I was thinking this was done to do queries and/or look at the schema. But it should only be done locally when working with the code.

Reply:
Very true, nice catch. Should not be checked into main branch. Removed